### ISSUE TITLE
Error: undefined Unable to resolve module AccessibilityInfo

### ISSUE THREAD

**Issue by rushibhumkar11** [Johncy1997/react-native-family-tree#41]
<img width="444" alt="Image" src="https://github.com/user-attachments/assets/f2c97aa0-3d69-42a9-a08c-2467dd7b41f9" />
